Gemfile

```diff
@@ -37,36 +25,29 @@ gem "jbuilder"
 # gem "bcrypt", "~> 3.1.7"
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
+gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 # Reduces boot times through caching; required in config/boot.rb
-gem "bootsnap", require: false
+gem 'bootsnap', require: false
+
+gem 'webpacker'
 
-# Use Sass to process CSS
-# gem "sassc-rails"
+gem 'rubocop', '>= 1.0', '< 2.0'
+
+gem 'stimulus-rails'
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
 
```

Comment:
- [ ] There should be `react-rails` gem in the `Gemfile`.

Reply:
addressed